### ISSUE TITLE
Exception with comment at the top of ns

### ISSUE THREAD

**Issue by yogsototh** [weavejester/cljfmt#23]
Here is the file I launch cljfmt on:

````
;; Break cljfmt 0.1.6
(ns break.cljfmt
  (:require [clojure.tools.logging :as log]))
````

And what I get:

````
java.lang.UnsupportedOperationException: null
 at rewrite_clj.node.comment.CommentNode.sexpr (comment.clj:6)
    rewrite_clj.node.protocols$eval5940$fn__5984$G__5946__5986.invoke (protocols.clj:9)
    rewrite_clj.node.protocols$eval5940$fn__5984$G__5945__5989.invoke (protocols.clj:9)
    clojure.core$juxt$fn__4211.invoke (core.clj:2440)
    rewrite_clj.node$value.invoke (node.clj:114)
    rewrite_clj.zip.base$value.invoke (base.clj:56)
    cljfmt.core$form_symbol.invoke (core.clj:121)
    cljfmt.core$inner_indent.invoke (core.clj:125)
    cljfmt.core$eval7950$fn__7952$fn__7954.invoke (core.clj:160)
    clojure.core$some_fn$sp2__6490.invoke (core.clj:6800)
    clojure.core$some_fn$spn__6527$fn__6528.invoke (core.clj:6817)
    clojure.core$some.invoke (core.clj:2515)
    clojure.core$some_fn$spn__6527.invoke (core.clj:6817)
    cljfmt.core$custom_indent.invoke (core.clj:177)
    cljfmt.core$indent_amount.invoke (core.clj:182)
    cljfmt.core$indent_line.invoke (core.clj:188)
    cljfmt.core$indent$fn__7981.invoke (core.clj:195)
    cljfmt.core$edit_all.invoke (core.clj:13)
    clojure.lang.AFn.applyToHelper (AFn.java:160)
    clojure.lang.AFn.applyTo (AFn.java:144)
    clojure.core$apply.invoke (core.clj:626)
    cljfmt.core$transform.doInvoke (core.clj:17)
    clojure.lang.RestFn.invoke (RestFn.java:464)
    cljfmt.core$indent.invoke (core.clj:195)
    cljfmt.core$reindent.invoke (core.clj:198)
    cljfmt.core$reformat_form.doInvoke (core.clj:208)
    clojure.lang.RestFn.invoke (RestFn.java:423)
    cljfmt.core$reformat_string.doInvoke (core.clj:211)
    clojure.lang.RestFn.invoke (RestFn.java:423)
    leiningen.cljfmt$reformat_string.invoke (cljfmt.clj:23)
    leiningen.cljfmt$valid_format_QMARK_.invoke (cljfmt.clj:27)
    clojure.lang.AFn.applyToHelper (AFn.java:156)
    clojure.lang.AFn.applyTo (AFn.java:144)
    clojure.core$apply.invoke (core.clj:626)
    clojure.core$partial$fn__4228.doInvoke (core.clj:2468)
    clojure.lang.RestFn.invoke (RestFn.java:408)
    clojure.core$complement$fn__4082.invoke (core.clj:1370)
    clojure.core$filter$fn__4264.invoke (core.clj:2601)
    clojure.lang.LazySeq.sval (LazySeq.java:40)
    clojure.lang.LazySeq.seq (LazySeq.java:49)
    clojure.lang.RT.seq (RT.java:484)
    clojure.core$seq.invoke (core.clj:133)
    clojure.core$empty_QMARK_.invoke (core.clj:5706)
    leiningen.cljfmt$check.doInvoke (cljfmt.clj:58)
    clojure.lang.RestFn.invoke (RestFn.java:425)
    clojure.lang.AFn.applyToHelper (AFn.java:156)
    clojure.lang.RestFn.applyTo (RestFn.java:132)
    clojure.core$apply.invoke (core.clj:626)
    leiningen.cljfmt$cljfmt.doInvoke (cljfmt.clj:79)
    clojure.lang.RestFn.invoke (RestFn.java:442)
    clojure.lang.Var.invoke (Var.java:388)
    clojure.lang.AFn.applyToHelper (AFn.java:160)
    clojure.lang.Var.applyTo (Var.java:700)
    clojure.core$apply.invoke (core.clj:626)
    leiningen.core.main$partial_task$fn__6094.doInvoke (main.clj:263)
    clojure.lang.RestFn.applyTo (RestFn.java:139)
    clojure.lang.AFunction$1.doInvoke (AFunction.java:29)
    clojure.lang.RestFn.applyTo (RestFn.java:137)
    clojure.core$apply.invoke (core.clj:626)
    leiningen.core.main$apply_task.invoke (main.clj:313)
    leiningen.core.main$resolve_and_apply.invoke (main.clj:319)
    leiningen.core.main$_main$fn__6160.invoke (main.clj:392)
    leiningen.core.main$_main.doInvoke (main.clj:385)
    clojure.lang.RestFn.invoke (RestFn.java:436)
    clojure.lang.Var.invoke (Var.java:388)
    clojure.lang.AFn.applyToHelper (AFn.java:160)
    clojure.lang.Var.applyTo (Var.java:700)
    clojure.core$apply.invoke (core.clj:624)
    clojure.main$main_opt.invoke (main.clj:315)
    clojure.main$main.doInvoke (main.clj:420)
    clojure.lang.RestFn.invoke (RestFn.java:482)
    clojure.lang.Var.invoke (Var.java:401)
    clojure.lang.AFn.applyToHelper (AFn.java:171)
    clojure.lang.Var.applyTo (Var.java:700)
    clojure.main.main (main.java:37)
````

**Comment by weavejester** [comment]
Fixed in 0.1.7.

**Comment by yogsototh** [comment]
Thanks!
